src/simulations/collect.jl

```diff
@@ -1,18 +1,21 @@
+get_data(a, s::Symbol) = getproperty(a, s)
+get_data(a, f::Function) = f(a)
+
 """
-Collects agent properties (fields of the agent object) into a dataframe.
+Collect agent properties into a dataframe. `properties` can have symbols (agent fields) or functions that take an agent as input.
```

Comment:
The documentation strings need to have a call signature statement, since in Julia a function name doesn't equate a call signature due to multiple dispatch

```
    collect_agent_data(model, properties, step = 0)
Collect...
```